mousestyles/data/__init__.py

```diff
@@ -62,3 +62,80 @@ def load_all_features():
     other_features = [x.iloc[:, -1] for x in data_frames]
     other_features.insert(0, all_features_df)
     return pd.concat(other_features, axis=1)
+
+
+def load_intervals(feature):
+    """
+    Parameters
+    ----------
+    feature: str
+             where feature is one of ["AS", "F", "IS", "M_AS", "M_IS", "W"]
+    Returns
+    -------
+    output type: DataFrame
+    description: All the data from the feature directory into one data frame
+                 columns are strain, mouse, day, start, stop
+    """
+    # get all file names
+    file_names = _os.listdir(_os.path.join(data_dir, "intervals", feature))
+    file_names = [name for name in file_names if len(name) > 10]
```

Comment:
Oh, maybe because I downloaded the zip data, so when I unzip it, there were some hidden file thats not .npy data. So I add it that to ensure I'm reading all the files I want. But I can delete that part if its not necessary.
and what do you mean by choices should be in brackets?
@dylandaniels